package.json

```diff
@@ -62,7 +58,8 @@
     "checkpoint-stream": "^0.1.1",
     "events-intercept": "^2.0.0",
     "extend": "^3.0.2",
-    "google-gax": "^1.7.5",
+    "google-auth-library": "^6.0.0",
```

Comment:
I don't like that this is explicit, but we can clean up later

Reply:
This was used directly somewhere and `eslint` yelled at me, so I added it here. Same with other dependencies added.